### PR TITLE
add sealer rollback

### PR DESCRIPTION
**Pull request by wizardlyk**
### Describe what this PR does / why we need it
add sealer rollback

### CODE REVIEW

File: cmd/sealer/cmd/cluster/rollback.go [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+		plugins = append(plugins, cf.GetPlugins()...)
+	}
+
+	runtimeConfig := &clusterruntime.RuntimeConfig{
+		Distributor:            distributor,
+		Plugins:                plugins,
+		ContainerRuntimeConfig: cluster.Spec.ContainerRuntime,
+	}
+
+	rollback, err := clusterruntime.NewInstaller(infraDriver, *runtimeConfig, clusterruntime.GetClusterInstallInfo(imageSpec.ImageExtension.Labels, runtimeConfig.ContainerRuntimeConfig))
```
[the remark this answers is not in the document]
Comment (wizardlyk) [reply cmd/sealer/cmd/cluster/rollback.go]: done

**Review by VinceCui — APPROVED** [review]
LGTM

File: cmd/sealer/cmd/cluster/rollback.go [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+			}
+
+			imageEngine, err := imageengine.NewImageEngine(options.EngineGlobalConfigurations{})
+			if err != nil {
+				return err
+			}
+
+			id, err := imageEngine.Pull(&options.PullOptions{
+				Quiet:      false,
+				PullPolicy: "missing",
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/rollback.go:73]: use constant here?

File: cmd/sealer/cmd/cluster/rollback.go [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+				Image:      args[0],
+				Platform:   "local",
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/rollback.go:75]: ditto

File: cmd/sealer/cmd/cluster/rollback.go (hunk 2 above)
Comment (wizardlyk) [reply cmd/sealer/cmd/cluster/rollback.go:65]: same problem in run.go，recommend to solve in one branch together。

File: cmd/sealer/cmd/cluster/rollback.go [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+			if err != nil {
+				return err
+			}
+
+			return rollbackCluster(newClusterfile, imageEngine, imageSpec)
+		},
+	}
+
+	rollbackFlags = &types.RollbackFlags{}
+	rollbackCmd.Flags().StringSliceVar(&rollbackFlags.AppNames, "apps", nil, "override default AppNames of sealer image")
```
[the remark this answers is not in the document]
Comment (Starnop) [reply cmd/sealer/cmd/cluster/rollback.go:105]: otherwise,LGTM

**Pull request merged**